tools/init2json.py

```diff
@@ -41,6 +44,31 @@ def parse_metadata_from_section(metadata_section):
     )
 
 
+def parse_contentnode_extras(config):
+    def is_tag_valid(tag):
+        return tag in ALLOWED_TAGS
+
+    section_names = []
+    for section in config.sections():
+        if section.startswith("contentnodeextras-"):
+            section_names.append(section)
+
+    tagged_node_ids = []
+    for name in section_names:
+        section = config[name]
+        # FIXME validate that the node_id is included
```

Comment:
This is a really good point, but I wouldn't worry about doing this validation here. It would be pretty expensive since we'd need to load up Kolibri and go through channel metadata to figure out if the node is anywhere in the tree we've selected! But perhaps we could add a QA script (ideally wired up to github actions) which does a test import of each of the collections we have? The code which actually installs a collection is in a good position for this kind of thing.

I made a task about that in Phabricator: https://phabricator.endlessm.com/T34345.

Reply:
Excellent, thanks for creating the follow-up. Sounds reasonable to not do the validation here.